.github/maven/settings.xml

```diff
@@ -0,0 +1,12 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <servers>
+        <server>
+            <id>gcs-maven-central-mirror</id>
+            <configuration>
+                <aether.connector.connectTimeout>60000</aether.connector.connectTimeout>
```

Comment:
```xml
<connectTimeout>60000</connectTimeout>
```

https://maven.apache.org/guides/mini/guide-resolver-transport.html#timeouts

Reply:
Let's try this. Thanks.